### ISSUE TITLE
Unable to view Web3 results in Ubuntu console, such as <web3.eth.getBlock('latest')>

### ISSUE THREAD

**Issue by keennay** [ethereum/web3.js#2759]
#### Expected behavior
Expecting to view basic information from a local (IpcProvider) Geth node, such as the latest synced block. This node has already been fully synced & properly showing data in console (geth attach) & Python.

#### Actual behavior
No data is outputted in console after running the *.js script. Returns to the console prompt.

#### Steps to reproduce the behavior
Code:

```
const Web3 = require('web3');
const net = require('net');

const web3 = new Web3(
new Web3.providers.IpcProvider('/ssd_nvme_geth/mainnet/geth.ipc', net));

web3.eth.getBlock('latest').then(console.log);
```

### Versions 
* OS: Ubuntu 18.04.2 LTS
* web3.js: 1.0.0-beta.53
* nodejs: v10.15.3
* ethereum node: Geth v1.9.0-unstable

**Comment by nivida** [comment]
Thanks for opening this issue but this bug got already fixed with PR https://github.com/ethereum/web3.js/pull/2758. (I will release it asap)

**Comment by keennay** [comment]
> Thanks for opening this issue but this bug got already fixed with PR #2758. (I will release it asap)

Thanks, confirmed working properly.
